src/main/webapp/profileScript.js

```diff
@@ -171,6 +181,35 @@ function addRouteDetails(newRoute) {
   return routeDetails;
 }
 
+function addRouteImageForm(container, newRoute) {
+  let imageForm = document.createElement("form");
+  imageForm.action = "/route-image";
+  imageForm.method = "POST";
+  imageForm.enctype = "multipart/form-data";
+  imageForm.id = "route-image-form";
+
+  let imageInput = document.createElement("input");
+  imageInput.type = "file";
+  imageInput.id = "route-image";
+  imageInput.name = "route-image";
+  imageInput.accept = "image/png, image/jpeg";
+
+  let imageName = document.createElement("input");
+  imageName.type = "text";
```

Comment:
This should probably be of type "hidden", seeing that the user should not edit this.

Reply:
I added "display: none" to the style file for profile page.